examples/amp-story/json/amp-story-auto-ads-0.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
We aren't actually using any of these json files, since we are using the ABE backend.

Reply:
Deleted.